### PR TITLE
[dunfell] Fix for ROS_DISTRO variable and meta-ros-python2 separation

### PR DESCRIPTION
**Pull request by shr-project**

### CODE REVIEW

File: meta-ros-common/conf/ros-distro/ros-distro.conf [hunk 1]
```diff
@@ -40,7 +40,7 @@ ROS_DISTRO_METADATA_VERSION_REVISION = "${ROS_DISTRO_METADATA_MILESTONE_NUMBER}"
 ROS_DISTRO_METADATA_VERSION = "${ROS_DISTRO_METADATA_VERSION_MAJOR}.${ROS_DISTRO_METADATA_VERSION_MINOR}.${ROS_DISTRO_METADATA_VERSION_MICRO}.${ROS_DISTRO_METADATA_VERSION_UPDATE}"
 ROS_DISTRO_METADATA_VERSION .= "-${ROS_DISTRO_METADATA_VERSION_REVISION}"
 
-ROS_DISTRO ??= "${@d.getVar('ROS2_DISTRO') if d.getVar('ROS2_DISTRO') != '' else d.getVar('ROS1_DISTRO')}"
+ROS_DISTRO ??= "${@d.getVar('ROS2_DISTRO') if d.getVar('ROS2_DISTRO', True) not in (None, '') else d.getVar('ROS1_DISTRO', True)}"
```
Comment (herb-kuta-lge) [inline meta-ros-common/conf/ros-distro/ros-distro.conf]: Isn't `True` the default the 2nd parameter of `d.getVar()`?

File: meta-ros-common/conf/distro/include/ros-common.inc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # OE DISTRO configuration settings common to both ROS 1 and ROS 2 distros.
 #
-# Copyright (c) 2019 LG Electronics, Inc.
```
Comment (herb-kuta-lge) [inline meta-ros-common/conf/distro/include/ros-common.inc:3]: Commit message: I've been using the convention of have a space between ROS and [12].

**Review by herb-kuta-lge — CHANGES_REQUESTED** [review]

File: meta-ros-common/conf/ros-distro/ros-distro.conf (hunk 1 above)
Comment (shr-project) [reply meta-ros-common/conf/ros-distro/ros-distro.conf]: Yes, I've added it when debugging why it didn't work and then forgot to drop it. Will do.

File: meta-ros-common/conf/distro/include/ros-common.inc (hunk 2 above)
Comment (shr-project) [reply meta-ros-common/conf/distro/include/ros-common.inc:3]: in 'ROS[12]_DISTRO' when I want to express ROS1_DISTRO or ROS2_DISTRO variable?

Comment (herb-kuta-lge) [reply meta-ros-common/conf/distro/include/ros-common.inc:3]: No, just in the commit message, eg, 

> and enable it by default for all ROS2 distributions as well as ROS1 noetic

-> and enable it by default for all ROS 2 distributions as well as ROS 1 noetic

**Pull request merged**